react-time-track/src/components/list-projects.js

```diff
@@ -1,7 +1,7 @@
 /** @jsx jsx */
 import React from "react";
 import { jsx } from "@emotion/core";
-import { navigate } from "@reach/router";
+import { navigate, Link } from "@reach/router";
```

Comment:
I think you should delete "navigate" because you don't use it here.

Reply:
Thanks, it will be deleted on the next PR